include/revng/Yield/Assembly/DisassemblerInstance.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <memory>
+
+#include "llvm/MC/MCAsmInfo.h"
+#include "llvm/MC/MCContext.h"
+#include "llvm/MC/MCDisassembler/MCDisassembler.h"
+#include "llvm/MC/MCInstPrinter.h"
+#include "llvm/MC/MCInstrInfo.h"
+#include "llvm/MC/MCObjectFileInfo.h"
+#include "llvm/MC/MCRegisterInfo.h"
+#include "llvm/MC/MCSubtargetInfo.h"
+
+#include "revng/Support/MetaAddress.h"
+#include "revng/Yield/Assembly/Assembly.h"
+
+class DisassemblerInstance {
```

Comment:
Maybe `LLVMDisassembler`?

Reply:
`LLVMDisassemblerInterface` then?

Reply:
Done: https://github.com/revng/revng/pull/173/commits/b6a22bf923ffda4d532db35491cfe67995392441 and https://github.com/revng/revng/pull/173/commits/d4ebf7ce88899ab242332c365e978077773f8e0b.